Resources/Locale/en-US/mind/role-types.ftl

```diff
@@ -0,0 +1,9 @@
+role-type-neutral-name = Neutral
```

Comment:
I think "Neutral" may be a little ambiguous. People like the Captain are not neutral, but rather "crew-aligned".

Neutral may cause confusion for people assuming they're allowed to just do anything, rather than doing the job NT is employing them for. Neutral may then make sense to keep for people like visitors, who aren't attached to the station (or maybe cognizine'd animals?).

Reply:
This is mostly an admin detail to decide. I have ran most parts of this by the admins already, and don't recall hearing objections to Neutral. That said, now is the final check to review everything, and if they decide to change it to some other name, then it will of course be changed. 
But it will have to be the same for everyone, the whole original concept of role types, back when they were introduced to the rules, was for them to be specific, consistent and simple. Creating more and more role types should be avoided (it is for that reason that I was hesitant to even add Altered Silicon, but it's kind of a UI organisation thing, unless we want to go back to using the separate Antag bool)

Reply:
After internal discussion, Neutral was renamed to Crew Aligned